drivers/serial/uart_stm32.c

```diff
@@ -1444,6 +1446,23 @@ static int uart_stm32_set_power_state(const struct device *dev,
 		/* Clear OVERRUN flag */
 		LL_USART_ClearFlag_ORE(UartInstance);
 		/* Leave UartInstance unchanged */
+
+		/* Configure dt provided device signals when available */
+		err = stm32_dt_pinctrl_configure(config->lp_pinctrl_list,
+						 config->lp_pinctrl_list_size,
+						 (uint32_t)UART_STRUCT(dev));
+		if (err < 0) {
+			return err;
+		}
+
+	} else {
+		/* Configure dt provided device signals when available */
+		err = stm32_dt_pinctrl_configure(config->pinctrl_list,
```

Comment:
```suggestion
		err = stm32_dt_pinctrl_configure(config->lp_pinctrl_list,
```

Reply:
At this point we need to provide active state pinmux configuration.
So I think this is correct.
I'll update comments though

Reply:
@alexanderwachter Can you review ?